examples/modules/arrowModuleClient1.wyv

```diff
@@ -0,0 +1,3 @@
+import modules.arrowModuleImplementation1
```

Comment:
Can you put these test files in tools/src/wyvern/tools/tests/modules?

Although the current project isn't completely consistent about this, I'd like to keep examples/ for files that do something interesting.  (Alternatively, you can work this test case into an example that does something interesting)